backend/python/app/tests/services/test_story_service.py

```diff
@@ -67,8 +73,60 @@ def test_get_stories_available_for_translation():
     pass
 
 
-def test_create_translation():
-    pass
+def test_create_translation(app, db, services):
+    alphanumeric = "abcdefghijklmnopqrstuvwxyz0123456789"
```

Comment:
Im seeing repeated code for generating translator, reviewer, and story. thoughts on making something a helper function to handle this? like `create_story_translation` in https://github.com/uwblueprint/planet-read/pull/183/files